pkg/tool/exec/exec.go

```diff
@@ -80,15 +80,19 @@ func (c *execCmd) Run(ctx *task.Context) (res interface{}, err error) {
 		err = cmd.Run()
 	}
 	update["success"] = err == nil
-	if err != nil {
-		if exit := (*exec.ExitError)(nil); errors.As(err, &exit) && captureErr {
+
+	if err == nil {
+		return update, nil
+	}
+
+	if exit := (*exec.ExitError)(nil); errors.As(err, &exit) {
```

Comment:
I've never seen this single-line style for As before, interesting :) seems fine to me to be clear.